example/lib/main.dart

```diff
@@ -1,61 +1,268 @@
-import 'package:flutter/material.dart';
-import 'dart:async';
+import 'dart:io';
 
-import 'package:flutter/services.dart';
 import 'package:ai_model_land/ai_model_land.dart';
+import 'package:ai_model_land/ai_model_land_lib.dart';
+import 'package:ai_model_land/modules/core/base_model.dart';
+import 'package:ai_model_land_example/singlton/ai_model_provider.dart';
+import 'package:ai_model_land_example/modalPage.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'addModelPage.dart';
+import 'package:provider/provider.dart';
 
 void main() {
   runApp(const MyApp());
 }
 
-class MyApp extends StatefulWidget {
+class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
   @override
-  State<MyApp> createState() => _MyAppState();
+  Widget build(BuildContext context) {
+    return MultiProvider(
```

Comment:
Its incorrect erchitecture of the example , pls look at this code- https://github.com/vlmoon99/flutterchain/tree/main/example  , and afetr this code take this exampels as the templete for the example - https://github.com/vlmoon99/cool_template_flutter

Reply:
How I'm you at this?

Reply:
> How I'm you at this?

Lol, I dont know what to say you, because Im really dont know why you participant in this review )